pyfarm/master/etc/master.yml

```diff
@@ -12,7 +12,206 @@
+# production.
+secret_key: pyfarm
+
+
+# The key used for signing the csrf token.
+#
+# **Production Note**: This value should be random, consistent between
+# frontends, and kept secret.  Do not use the value below for
+# production.
+csrf_session_key: pyfarm
+
+
+# The prefix of the URL from which the API will operate on.  This should
+# not generally be changed unless you are operating different versions
+# of the API at the same time from one web server.
+api_prefix: /api/v1
+
+
+# The URL template we use to communicate with the agent.
+agent_api_url_template: http://${host}:${port}/api/v1
+
+
+
+# Enables or disable the login functionality.  This can be used when
+# debugging or doing development but should not be changed for
+# production.
+login_disabled: false
+
+
+# The amount of time the 'remeber me' cookie should persist.  The keys
+# and values here are passed into a `timedelta` object as keywords.
+cookie_duration:
+  hours: 16
+
+
+# When true json output from te APIs will be reformatted to
+# be more human readable.
+pretty_json: false
+
+
+# When true all SQLAlchemy queries will be echoed.  This is useful
+# for debugging the SQL statements being run and to get an idea of
+# what the underlying ORM may be doing.
+echo_sql: false
+
+
+# When true the application will automatically create users in
+# PyFarm's database if they do not exist already.  Setting this
+# to false will cause an exception to be raised if the user in
+# question does not exist.
+autocreate_users: true
+
+
+# When autocreating users, use this email address as a template.  For example:
+#   {username}@example.com
+# Not setting this value and setting `autocreate_users` to true will result
+# in a users's email address not being set on a newly created user.
+autocreate_user_email: null
+
+
+# When provided an integer this many seconds will elapse after a job
+# has completed before it is deleted.
+default_job_delete_time: null
+
+
+# The format for timestamps in the user interface.
+timestamp_format: "YYYY-MM-DD HH:mm:ss"
+
+
+# The directory to store updates for agents.  This will use `temp` above
+# as the base directory.
+agent_updates_dir: ${temp}/pyfarm-updates
+
+
+# Optional directory to serve GET requests for agent updates
+# from.  This is different from `agent_updates_dir` in that it's
+# only used when an agent is requested a file to update from.  This
+# can be useful when you're caching requests or doing something with
+# the update files prior to them being requested by the agent.
+agent_updates_webdir: null
+
+
+# The directory to store downloaded logs in.
+#
+# **Production Note**: For production it's probably best if these are kept
+# in a persistent location rather than $temp.
+tasklogs_dir: ${temp}/task_logs
+
+
+# The address the Flask application should listen on.  By default
+# this will only listen locally but could be changed to listen on
+# a specific adatper or `0.0.0.0` for all addresses.
```

Comment:
This should point out that this setting is only for flask's standalone test webserver